src/pages/Home/index.js

```diff
@@ -79,6 +79,9 @@ function Home() {
           setFilms={setFilms}
         />
       )}
+      {/*<div>
```

Comment:
Excuse me miss, What component will be this one?

Reply:
It is one of my issues on Sprint 2. But I've not added the code yet. Because I only have the main structure and it is not part of this issue.